src/Grid/parse_the_grids.jl

```diff
@@ -0,0 +1,120 @@
+function parse_the_grids(path::String="/tmp/grid.toml")
```

Comment:
Since this function is exported users will see it. Hence a doc string should be added for users.